src/clarity-angular/layout/vertical-nav/vertical-nav-group.ts

```diff
@@ -102,26 +92,20 @@ export class VerticalNavGroup implements AfterContentInit, OnDestroy {
         }
     }
 
-    @Output("clrVerticalNavGroupExpandedChange")
-    expandedChange: EventEmitter<boolean> = new EventEmitter<boolean>(true);
-
-    private _active: boolean = false;
-
-    get active(): boolean {
-        return this._active;
-    }
-
-    @Input("clrVerticalNavGroupActive")
-    set active(value: boolean) {
+    @Input("clrVerticalNavGroupExpanded")
```

Comment:
Once again, please don't document this. I want to wait and see if there is really a need for a non-standard `ifExpanded` binding. I know the theoretical use case, but I want to see if people actually stumble on it.